wagtail/wagtailadmin/static_src/wagtailadmin/images/logo-eyeclosed.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="430" height="537" viewBox="0 0 430 537"><path d="M341 77.4c.2 0 .4 0 .5-.3l.6-.6.5-.6.4-1s.2 0 .4-.3c0-.3.4-.5.6-.8 0-.2.4-.5.7-.7l.7-.4c.4 0 .5-.4 1-.6l.6-.4c.2 0 .3-.2.5-.3l.7-.4.5-.4.7-.4.7-.5.8-.4.5-.3c.3-.3.5-.3 1-.5l.6-.4.6-.2s.4 0 .7-.3l.7-.3c.4 0 .7-.2 1-.4h.7l.5-.3h.6l.7-.4h.6l.7-.2h4.5c.4.2.7.2 1 .2h.8c.3 0 .4 0 .6.2h1l.5.2h.5c.3 0 .5.4.7.4h.7c0 .2.4.2 1 .4l.7.3h.8l.7.4h.5" stroke-opacity="null" stroke-width="12" stroke="#231F20" fill="none"/></svg>
+<svg xmlns="http://www.w3.org/2000/svg" width="430" height="537" viewBox="0 0 430 537"><path d="M343 77.2c14-16 33.8-7 33.8-7" fill="none" stroke="#231f20" stroke-width="12"/></svg>
```

Comment:
It's not immediatly clear to me why this has changed, could you explain the difference?

Reply:
The original one was drawn freehand in an online svg editor which looked bad and was poorly optimised. The new one is a simple curve, so smaller file size and (more importantly) looks much better. It's not so noticeable in the nav-menu as the size of the logo is so small, but it was very apparent on this page where the size is larger.

It's more noticeable if you look at a side-by-side comparison: https://cdn.pbrd.co/images/GHr1ZGM.png

Reply:
Makes sense, thank you for the details!